src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/WhereCQLClauseAnalyzer.java

```diff
@@ -70,6 +70,8 @@ AnalysisStrategy getStrategy(WhereCQLClauseAnalyzer analyzer) {
                     analyzer::checkNoFilters,
                     analyzer::checkAllColumnsExist,
                     analyzer::checkNonPrimaryKeyFilters,
+                    analyzer::checkOnlyEqUsage,
```

Comment:
Fine for now but in future we probably will want to pre-create these `List`s to reduce garbage generation.
(so no change proposed for this PR, just mention as future improvement)